src/ripple/overlay/impl/ZeroCopyStream.h

```diff
@@ -56,7 +56,7 @@ class ZeroCopyInputStream
     bool
     Skip (int count) override;
 
-    std::int64_t
+    google::protobuf::int64
```

Comment:
There are 3 other places in this file that should be consistent with this type:
1. Line 41: type of `count_`
2. Line 163: return type of another `ByteCount`.
3. Line 145: type of another `count_`.

I don't have a strong rationale for what type they should be.  Just that they should all be the same.